plugins/devtools-common/README.md

```diff
@@ -0,0 +1,3 @@
+# DevTools Common
+
+Common types and permissions for the DevTools plugin.
```

Comment:
this will need install instructions as well?  or does installing the backend install this by default?

if not, i guess we could either say how to install this here, and/or in the backend and front install instructions, just have two calls to installing plugins, just that you install both the backend and the common in the backend, and same in the frontend.

i don't recall offhand what pattern other apps which have a separate package use!

Reply:
would be good to link this back to the frontend plugin as well, as you may not need to provide full context for what this does, but it should link to what it's a dependency of.

Reply:
So this is a pretty common pattern that you'll see with a lot of newer-ish plugins, we started this with the Azure DevOps plugin actually so it's been around for a maybe 18 months or so. These `-common` packages usually contain Types and utility functions. Traditionally we have not documented these at all as there is really no need for Integrators to use these and when they do they get documented in the frontend or backend where needed

Now with the advent of the Permissions Framework you'll see all of that in these. Which raises the question how do we actually document that part? That I don't know and feel it should be the subject for a different PR, just trying to get this one done as it's very old now, that's mostly on me to be clear.